### PR TITLE
Fixing fhc git prop set

### PR DESCRIPTION
**Pull request by jasonmadigan**
`fhc git props` & `fhc git set` have been broken with a while, this patches and fixes the commands.

Some commands to test with below.

Reading git props for an app:

```
fhc git props 6magpwla4zuxssywj77u6o2x
```

Setting a default branch for an app:

```
fhc git set 6magpwla4zuxssywj77u6o2x branch master
```

### CODE REVIEW

**Comment by jasonmadigan** [comment]
Fixed this up a few months back, time to land it.

File: lib/cmd/common/git.js [hunk 1]
```diff
@@ -55,7 +58,7 @@ function props(appId, cb) {
 // Git set properties
 // Note: although we're just setting scm related properties, the app update endpoint expects the full config object..
 function set(appId, k, v, cb) {
-  common.readApp(appId, function (err, cfg) {
+  common.readAppWithoutProject(appId, function (err, cfg) {
```
Comment (fh-qube) [inline lib/cmd/common/git.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* There should be an associated fhcap PR with this change - please provide a link to this
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.

In order to update fhcap with your component change please execute:
`fhcap component update fh-fhc 2.7.1 277`

**Pull request closed without merging**